scripts/gen_circleci_config.py

```diff
@@ -68,6 +68,14 @@ def check(name: str, command: str, paths: t.Set[str]) -> None:
     )
 
 
+def gen_build_docs(template: dict) -> None:
```

Comment:
does this need to be called from somewhere?

Reply:
no it's called automatically by the boilerplate code at the bottom of the script